lib/ProductOpener/Products.pm

```diff
@@ -220,6 +216,21 @@ sub init_product($) {
 	if ($creator eq 'kiliweb') {
 		if (defined param('cc')) {
 			$country = param('cc');
+			
+			# 2019 06 01: also try to change the language according to country
```

Comment:
Is there a specific reason why this comment is different to this? https://github.com/openfoodfacts/openfoodfacts-server/blob/5430048b2d138e123098bbaed8b29b660d66859a/cgi/product_jqm_multilingual.pl#L142

Also, product_jqm_multilingual.pl checks for `param('cc') ne "fr"`. This most likely won't make a difference, because no override for `fr` is defined, but the logic might be easier to understand if it's the same for both files.